RUDP/src/net/rudp/ReliableSocket.java

```diff
@@ -1220,7 +1222,11 @@ private void connectionFailure()
                     synchronized (_recvQueueLock) {
                         _recvQueueLock.notify();
                     }
-
+                    
+                    synchronized (_unackedSentQueue) {
+                    	_unackedSentQueue.notifyAll();
```

Comment:
Is there a specific reason we need this twice? We're doing the same thing just a few lines above?